src/main/java/com/onarandombox/MultiverseCore/configuration/MVConfigNodes.java

```diff
@@ -0,0 +1,170 @@
+            .comment("#                    █▀▄▀█ █░█ █░░ ▀█▀ █ █░█ █▀▀ █▀█ █▀ █▀▀   █▀▀ █▀█ █▀█ █▀▀                      #")
+            .comment("#                    █░▀░█ █▄█ █▄▄ ░█░ █ ▀▄▀ ██▄ █▀▄ ▄█ ██▄   █▄▄ █▄█ █▀▄ ██▄                      #")
+            .comment("#                                                                                                  #")
+            .comment("#                                                                                                  #")
+            .comment("#    WIKI:        https://github.com/Multiverse/Multiverse-Core/wiki                               #")
+            .comment("#    DISCORD:     https://discord.gg/NZtfKky                                                       #")
+            .comment("#    BUG REPORTS: https://github.com/Multiverse/Multiverse-Core/issues                             #")
+            .comment("#                                                                                                  #")
+            .comment("#                                                                                                  #")
+            .comment("#    Each option in this file is documented and explained here:                                    #")
+            .comment("#     ==>  https://github.com/Multiverse/Multiverse-Core/wiki/config.yml                           #")
+            .comment("#                                                                                                  #")
+            .comment("#                                                                                                  #")
+            .comment("#    New options are added to this file automatically. If you manually made changes                #")
+            .comment("#    to this file while your server is running, please run `/mv reload` command.                   #")
+            .comment("#                                                                                                  #")
+            .comment("####################################################################################################")
+            .comment("")
+            .comment("")
+            .build());
+
+//    private static final MVCommentedNode WORLD_HEADER = node(MVCommentedNode.builder("world")
+//            .comment("")
+//            .comment("")
+//            .build());
+
+    public static final MVValueNode<Boolean> ENFORCE_ACCESS = node(MVValueNode.builder("world.enforce-access", Boolean.class)
+            .comment("This setting will prevent players from entering worlds they don't have access to.")
+            .comment("If this is set to false, players will be able to enter any world they want.")
+            .comment("If this is set to true, players will only be able to enter worlds they have")
+            .comment("the `mv.access.<worldname>` permission.")
+            .defaultValue(false)
+            .build());
```

Comment:
Consider using a lambda function parameter to reduce the need to know about `MVValueNode.builder` and `MVCommentedNode.builder`. For example `valueNode("world.enforce-access", Boolean.class, node -> node.comment...` This removes the need to call `build` at the end as well.